R/app_server.R

```diff
@@ -171,6 +171,19 @@ app_server <- function(input, output, session) {
   }) %>%
     bindEvent(selected_pkg$decision(), selected_pkg$overall_comment_added())
   
+  session$userData$user_report <- reactiveValues()
+  observe({
+    req(user$name)
+
+    # retrieve user data, if it exists.  Otherwise use rpt_choices above.
+    session$userData$user_report$user_file <- file.path(system.file("report_downloads", package = "riskassessment"), glue::glue("report_prefs_{user$name}.txt"))
```

Comment:
I was thinking more about the app storage, and I think the report specifications need to be stored in the root directory like the other data. If an organization has multiple implementations of the package (say for different projects) the current storage would cause all projects to share the same storage location which is not desirable.

Reply:
This directory has been moved to the db-config.yml file